rest_api/sawtooth_rest_api/rest_api.py

```diff
@@ -157,11 +175,12 @@ def main():
             log_configuration(log_dir=log_dir, name="sawtooth_rest_api")
         init_console_logging(verbose_level=opts.verbose)
 
+        host, port = rest_api_config.bind[0].split(":")
```

Comment:
There doesn't appear to be anything that validates that these two parts are there.  The `int(port)` will fail with an Error, that wont reflect back on the fact that port is missing or wrong.